.github/workflows/system-tests.yml

```
@@ -274,6 +274,20 @@ jobs:
       artifact_retention_days: ${{ inputs.artifact_retention_days }}
       unique_id: ${{ needs.compute_parameters.outputs.unique_id }}
 
+  stream_processing_offload:
```

To be consistent with other jobs 


```suggestion
  run_stream_processing_offload:
```

(yes, I should have made that comment for `external_processing` 🤦 )

---

Do I also apply the comment for `external_processing` or will I do it in another PR ?